sources/shared-defaults/nvidia-k8s-device-plugin-cdi.toml

```diff
@@ -0,0 +1,2 @@
+[settings.kubelet-device-plugins.nvidia]
+device-list-strategy="cdi-cri"
```

Comment:
You'll need to write a migration for whenever the `cdi-cri` enum variant is supported - it'll happen when you upgrade the settings SDK crates.

The new value will have to be removed on downgrade into older versions and set to something else, or `apiserver` will fail to deserialize settings from the datastore.